pkg/util/dtotel/traces_test.go

```diff
@@ -36,8 +36,13 @@ func TestStartSpan(t *testing.T) {
 	t.Run("nil tracer", func(t *testing.T) {
 		ctx, span := StartSpan(context.Background(), trace.Tracer(nil))
 
-		assert.IsType(t, noopSpan{}, span)
 		assert.Equal(t, context.Background(), ctx)
+
+		assert.IsType(t, noopSpan{}, span)
+		noopSpan, ok := span.(noopSpan)
+		require.True(t, ok)
+		assert.Contains(t, noopSpan.spanTitle, "dtotel.TestStartSpan.func")
```

Comment:
Equals would be better? What if somebody unintentionally changes the depth? the unit test would still work